modules/claims_api/spec/requests/v2/veterans/rswag_disability_compensation_request_spec.rb

```diff
@@ -93,6 +93,117 @@
           end
         end
       end
+
+      describe 'Getting an unauthorized reponse' do
+        response '401', 'Unauthorized' do
+          let(:scopes) { %w[system/claim.read system/claim.write] }
+
+          schema JSON.parse(Rails.root.join('spec', 'support', 'schemas', 'claims_api', 'v2', 'errors',
+                                            'default.json').read)
+
+          let(:data) do
+            temp = Rails.root.join('modules', 'claims_api', 'spec', 'fixtures', 'v2', 'veterans',
+                                   'disability_compensation', 'form_526_json_api.json').read
+            temp = JSON.parse(temp)
+            temp
+          end
+
+          before do |example|
+            stub_poa_verification
+            stub_mpi
+            # skip ccg authorization to fail authorization
+            VCR.use_cassette('claims_api/disability_comp') do
+              submit_request(example.metadata)
+            end
+          end
+
+          after do |example|
+            example.metadata[:response][:content] = {
+              'application/json' => {
+                example: JSON.parse(response.body, symbolize_names: true)
+              }
+            }
+          end
+
+          it 'returns a 401 response' do |example|
+            assert_response_matches_metadata(example.metadata)
+          end
+        end
+      end
+
+      describe 'Getting an unprocessable entity response' do
+        response '422', 'Unprocessable entity' do
+          let(:scopes) { %w[system/claim.read system/claim.write] }
+
+          schema JSON.parse(Rails.root.join('spec', 'support', 'schemas', 'claims_api', 'v2', 'errors',
+                                            '422.json').read)
+
+          # Build the dropdown for examples
+          def append_example_metadata(example, response)
+            example.metadata[:response][:content] = {
+              'application/json' => {
+                examples: {
+                  example.metadata[:example_group][:description] => {
+                    value: JSON.parse(response.body, symbolize_names: true)
+                  }
+                }
+              }
+            }
+          end
+
+          def make_request(example)
+            stub_poa_verification
+            stub_mpi
+
+            mock_ccg(scopes) do
+              VCR.use_cassette('claims_api/disability_comp') do
```

Comment:
So, I moved this vcr to the top so that we can use it for all the endpoints tested, and that worked. It would be nice if that worked for your and Matt's branches.

Reply:
yep good call